.github/workflows/docs.yml

```
@@ -0,0 +1,36 @@
+name: Publish Docs
+on:
+  push:
+    branches: ["master", "mkdocs-unify-docs"]
```

Can we remove the (temporary?) branch name after testing is successful?
```suggestion
    branches: ["master"]
```

---

Yeah, should be safe to remove that. I added it to build on my fork for the demo.